cypress/unit/store-getter-getHeuristic.cy.js

```diff
@@ -0,0 +1,45 @@
+import { getters } from "~/store";
+
+describe("The column-linking-table component", () => {
```

Comment:
```suggestion
describe("The getHeuristic getter", () => {
```
Did you mean this?

Reply:
@surchs  I should say it does! Thanks for catching that.